docroot/themes/custom/mass_theme/overrides/css/datatables.css

```
@@ -3,7 +3,8 @@ table.dataTable {
   min-width: unset;
 }
 
-table.dataTable thead {
+table.dataTable thead,
+table.dataTable thead tr>.dtfc-fixed-start {
```

```suggestion
table.dataTable thead tr>.dtfc-fixed-start {
```
where is this class being defined?
`.dtfc-fixed-start`

---

I don't see this class being used in the markup

---

Please disregard this comment - these are datable classnames https://datatables.net/extensions/fixedcolumns/examples/initialisation/left_right_columns.html